bin/improver-tests

```diff
@@ -158,6 +158,20 @@ function improver_test_unit {
     echo_ok "Unit tests"
 }
 
+function improver_test_recreate_checksums {
+    # Recreate checksums for acceptance test data.
+    LC_FORMER=${LC_ALL:-}
+    export LC_ALL=C
+    cd $IMPROVER_ACC_TEST_DIR
```

Comment:
Suggest using `pushd` and `popd` rather than `cd -` history lookup to be clearer about the intention here.
`pushd` and `popd` are bash specific rather than POSIX sh, but this script is already bash specific, so that's not a problem.

Reply:
Done.